File: 0268-missing-number/0268-missing-number.java
class Solution {
    public int missingNumber(int[] nums) {
        Arrays.sort(nums);
        int length = nums.length;
        int i=1;
         if (nums[0] != 0) {
                return 0;
            }
        for(;i<nums.length;i++){
            if(nums[i]-nums[i-1]!=1){
                return nums[i]-1;
            }
        }
    return length;
    }
}

File: 0268-missing-number/0268-missing-number.py
class Solution:
    def missingNumber(self, nums: List[int]) -> int:
        n = len(nums)

        actualSum = (n*(n+1))/2
        expectedSum = 0

        for i in range(n):
            expectedSum+=nums[i]

        return int(actualSum-expectedSum)
